### PR TITLE
Audit/market overview

### PR DESCRIPTION
**Pull request by giacomolicari**

### CODE REVIEW

File: src/containers/MarketListPage/index.js [hunk 1]
```diff
@@ -9,6 +9,8 @@ import { getDefaultAccount } from 'selectors/blockchain'
 
 import { requestMarkets } from 'actions/market'
 
+const config = require('config.json')
```
Comment (andre-meyer) [inline src/containers/MarketListPage/index.js]: why not import?

File: src/components/MarketList/index.js [hunk 2]
```diff
@@ -121,14 +121,14 @@ class MarketList extends Component {
...
-          <div className="info__group col-md-3">
+          {/* <div className="info__group col-md-3">
             <div className="info__field">
               <div className="info__field--icon icon icon--currency" />
               <div className="info__field--label">
                 <CurrencyName collateralToken={market.event.collateralToken} />
               </div>
             </div>
-          </div>
+          </div>*/}
```
Comment (andre-meyer) [inline src/components/MarketList/index.js]: Can you remove commented out sections please? They will be in our git log if they're ever needed again

File: src/containers/MarketListPage/index.js (hunk 1 above)
Comment (giacomolicari) [reply src/containers/MarketListPage/index.js]: config.json contains a JSON dictionary, not a Javascript object. Is there a way to use Import with json files?

Comment (andre-meyer) [reply src/containers/MarketListPage/index.js]: What happens if you try `import config from 'config.json'`?

**Pull request closed without merging**